.claude/hooks/README.md

```
@@ -0,0 +1,62 @@
+1. **Configure Claude Code settings** (`.claude/settings.json`):
+
+```json
+{
+  "statusLine": {
+    "type": "command",
+    "command": ".claude/hooks/message-counter.sh"
+  }
+}
+```
+
+2. **Restart Claude Code** to activate the status line
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)